src/docs/user/references/JobPlannerServiceReference.adoc

```diff
@@ -21,6 +21,23 @@ One can see the cron expression "0 23 0 ? * MON,TUE,WED,THU,FRI *", which follow
 <<_job_planner_cron_expression_syntax, Quartz Cron Expression Syntax>> section. The cron expression in this example
  executes 23 minutes past every hour, on working days (Monday to Friday).
 
+==== Inclusion Calendar
+
+The purpose of the inclusion calendar is to use ICS file to launch workflows during events. For each events a pre-defined action will be executed an will define when the workflow will be executed.
+
+===== Available inclusion actions
+
+[cols="1,1", options="header"]
+|===
+
+|Inclusion action
+|Description
+
+|EXECUTE_AT_START
+|The workflow will be executed at each event start.
```

Comment:
The workflow is executed at the scheduled time. The action are not selected by the user at runtime but during the workflow definition.